docker-compose.storage-squid.yml

```
@@ -0,0 +1,79 @@
+version: '3'
+
+services:
+  squid_db:
+    container_name: squid_db
+    hostname: squid-db
+    image: postgres:14
+    restart: unless-stopped
+    environment:
+      POSTGRES_DB: ${SQUID_DB_NAME}
+      POSTGRES_PASSWORD: ${SQUID_DB_PASS}
+    networks:
+      - joystream
+    ports:
+      - '${SQUID_DB_PORT}:${SQUID_DB_PORT}'
```

```suggestion
      - '127.0.0.1:${SQUID_DB_PORT}:${SQUID_DB_PORT}'
```

To prevent the db being exposed on public interface?

---

Addressed.